db/seeds/posts/moderator.html

```diff
@@ -1,6 +1,6 @@
 <p>So you&#39;ve been given moderator status on a Codidact site. Welcome! This is what that means for you.</p>
 <h2 id="your-community-is-your-community">Your community is your community</h2>
-<p>Codidact sites are community-run. Codidact is here to support you and your community, but you make the decisions about how you run things yourselves. We&#39;ll provide our advice and guidance and let you know what we think works well, but if your community wants to go in a different direction, that&#39;s up to you!</p>
+<p>Codidact communities are community-run. Codidact is here to support you and your community, but you make the decisions about how you run things yourselves. We&#39;ll provide our advice and guidance and let you know what we think works well, but if your community wants to go in a different direction, that&#39;s up to you!</p>
```

Comment:
Non-replacement would make more sense to me here...

Reply:
Maybe: At Codidact, we believe that communities should run themselves.  Codidact is here to... (etc).

Reply:
Regardless, I went and reverted my changes to this file - if there is a consensus to override to communities happy to revert my reversion